tron/mesos.py

```diff
@@ -362,6 +383,23 @@ def submit(self, task):
 
         mesos_task_id = task.get_mesos_id()
         self.tasks[mesos_task_id] = task
+        env = task.get_config()['environment']
+        clusterman_resource_str = env.get('CLUSTERMAN_RESOURCES')
+        clusterman_metrics, _ = get_clusterman_metrics()
+        if clusterman_resource_str and clusterman_metrics:
+            clusterman_resources = json.loads(clusterman_resource_str)
+            aws_region = env.get('EXECUTOR_AWS_REGION')
+            cluster = env.get('EXECUTOR_CLUSTER', env.get('PAASTA_CLUSTER'))
+            pool = env.get('EXECUTOR_POOL', env.get('PAASTA_POOL'))
+            metrics_client = clusterman_metrics.ClustermanMetricsBotoClient(
+                region_name=aws_region,
+                app_identifier=f'tron_{cluster}_{pool}',
+            )
+            with metrics_client.get_writer(
+                clusterman_metrics.APP_METRICS, aggregate_meteorite_dims=True
+            ) as writer:
+                for metric_key, desired_quantity in clusterman_resources.items():
+                    writer.send((metric_key, int(time.time()), desired_quantity))
```

Comment:
ultra-mega-nit: the metric values can be anything -- doesn't have to be desired quantity

Reply:
fixed